### PR TITLE
Migrated to py3, added installation script and updated README.md

### PR DESCRIPTION
**Pull request by RajSolai**
- Application is named as 'easydnscli'
- Version 1.0
- Migrated from python2 to python3
- Added an installation script
  - Creates an executable for the platform
  - The executable is moved/installed to '/usr/local/bin'
  - Installation currently supports Linux and MacOS but it can build for windows also
- Changes in README
  - The Readme author section is updated
  - The installation sample is given

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -2,28 +2,19 @@
...
-  -f CONF, --file=CONF  configuration file containing easyDNS API details
-  -c, --create          create new record
-  -u, --update          update existing record to new IP address
-  -H HOSTNAME, --hostname=HOSTNAME
-                        specify short hostname without domain name part. e.g:
-                        www for www.example.com
-  -a IPADDR, --address=IPADDR
-                        specify IP address for the hostname
-```
+### Requirements
```
Comment (tuladhar) [inline README.md]: Let's add EasyDNS API in the requirement section.

File: README.md [hunk 2]
```diff
@@ -2,28 +2,19 @@
...
+- [git](https://git-scm.com/downloads) 
+- [Python 3](https://www.python.org/downloads/)
 
 ### INSTALLATION
 ```
-❯❯❯ pip install requests
-❯❯❯ curl -sO https://github.com/tuladhar/easydns-restapi-cli/blob/master/easydns-restapi-cli.py
-❯❯❯ install -m 755 easydns-restapi-cli.py /usr/local/bin
+$ git clone https://github.com/tuladhar/easydns-restapi-cli
+$ cd easydns-restapi-cli && install.sh
```
Comment (tuladhar) [inline README.md]: Let's use Makefile instead of install.sh

**Review by tuladhar — CHANGES_REQUESTED** [review]

**Comment by tuladhar** [comment]
Thanks @RajSolai. Instead of install.sh let's use of Makefile, here are few links you can use as a reference:
- https://github.com/tuladhar/set-brightness/blob/main/Makefile

File: README.md (hunk 1 above)
Comment (RajSolai) [reply README.md]: will do it , Nice

File: README.md (hunk 2 above)
Comment (RajSolai) [reply README.md]: Working on it !

**Review by RajSolai — COMMENTED** [review]
- Usage of Makefile for Installation
  - Makefile is created
  - executable is compiled
  - bin folder is created
- Updates in the README
  - added EASYDNS API to Requirement section
  - changes in the installtion sample

File: README.md [hunk 3]
```diff
@@ -2,28 +2,21 @@
...
+- [EasyDNS API](https://docs.sandbox.rest.easydns.net/)
 
 ### INSTALLATION
 ```
-❯❯❯ pip install requests
-❯❯❯ curl -sO https://github.com/tuladhar/easydns-restapi-cli/blob/master/easydns-restapi-cli.py
-❯❯❯ install -m 755 easydns-restapi-cli.py /usr/local/bin
+$ git clone https://github.com/tuladhar/easydns-restapi-cli && cd easydns-restapi-cli
+$ sudo make install
+$ easydnscli: installed
```
Comment (tuladhar) [inline README.md]:
```suggestion
```

**Review by tuladhar — CHANGES_REQUESTED** [review]

**Comment by tuladhar** [comment]
@RajSolai Let's keep sample config file intact. I've requested for few more changes, besides that looks good to merge.

**Comment by tuladhar** [comment]
@RajSolai Can you also document how you generate the `easydnscli` binary, and let's add that to README as well.

File: README.md (hunk 3 above)
Comment (RajSolai) [reply README.md]: Sorry for removing the sample config file , What can should I change ? The program name ??

**Comment by RajSolai** [comment]
Sure I'll add the "build from source" section in the README

File: README.md (hunk 3 above)
Comment (tuladhar) [reply README.md]: Looks like this line is the output of the install command, so we can omit that.

Comment (RajSolai) [reply README.md]: Sure I'll omit that one

File: sample-easydns.conf [hunk 4]
```diff
@@ -0,0 +1,9 @@
+{
+	"domain": "XYZ",
+	"token": "XYZ",
+	"key": "XYZ",
+	"endpoint": "http://sandbox.rest.easydns.net",
+	"format": "json",
+	"delay": 5,
+	"ttl": 600
+}
```
Comment (tuladhar) [inline sample-easydns.conf]:
```suggestion
}

```

**Review by tuladhar — CHANGES_REQUESTED** [review]

File: sample-easydns.conf (hunk 4 above)
Comment (tuladhar) [reply sample-easydns.conf]: Add EOF.

Comment (RajSolai) [reply sample-easydns.conf]: I'll check that !

Comment (RajSolai) [reply sample-easydns.conf]: Can you be more specific how can I implement the End of File ?

Comment (tuladhar) [reply sample-easydns.conf]: It's simple, just add a new line at the end of the file. Github complains that's why. I've also suggested the change.

Comment (RajSolai) [reply sample-easydns.conf]: Okay fine , I'll do it

Comment (RajSolai) [reply sample-easydns.conf]: Done adding !

**Review by tuladhar — APPROVED** [review]

**Comment by tuladhar** [comment]
Looking great. Thanks @RajSolai. Merging it.

**Pull request merged**